cmd/state/internal/cmdtree/ppm.go

```diff
@@ -33,14 +36,18 @@ func newPpmCommand() *captain.Command {
 					return printDefault()
 				}
 			}
+			err := runner.StartConversionFlowIfNecessary()
+			if err != nil {
+				return err
+			}
 			return printMain()
 		},
 	)
 
 	var children []*captain.Command
-	children = addPackagesCommands(children)
+	children = addPackagesCommands(runner, children)
```

Comment:
Don't pass runners around. Runners are top level.

Reply:
As mentioned earlier, I would like to wait for @MDrakos to merge his blocking PR before I change this file, and keep the changes to it to a minimum to simplify merging.  I just added some placeholder to the commands that need the conversion logic, but that is temporary, of course...